dev/run-test-container.sh

```diff
@@ -0,0 +1,4 @@
+ROOT_DIR=$(git rev-parse --show-toplevel)
+docker build -f "${ROOT_DIR}/dev/Dockerfile.test" -t mlflow-test-env "$ROOT_DIR"
```

Comment:
```suggestion
DOCKER_BUILDKIT=1 docker build -f "${ROOT_DIR}/dev/Dockerfile.test" -t mlflow-test-env "$ROOT_DIR"
```

Can we run `docker build` with `DOCKER_BUILDKIT=1`? In my Ubuntu machine, buildkit is not enabled and `.dockerignore` in the repository root was used when I ran this script.

Reply:
sure, good point